cmake/AutoGeneratedTargets.cmake

```diff
@@ -1,11 +1,11 @@
-# Autogenerated by ./script/cmake/autogen on date Fri May 25 12:43:26 DST 2018
+# Autogenerated by ./script/cmake/autogen on date Sun Jul 15 21:23:58 CEST 2018
 
 mk_declare_example("example/ffi" "ndt")
 mk_declare_example("example/ffi" "web_connectivity")
+mk_declare_example("example/nettest" "ndt")
+mk_declare_example("example/nettest" "web_connectivity")
 mk_declare_example("example/nettests" "multithread")
 mk_declare_example("example/nettests" "simple")
-mk_declare_example("example/swig" "ndt")
-mk_declare_example("example/swig" "web_connectivity")
```

Comment:
This PR removes the `swig` API. The `nettest` API is designed to be easily integrable using SWIG for both Java and Golang (even though we're not planning on doing that _now_ for golang). These changes reflect the fact that the `swig` API examples are gone and the new `nettest` API examples have been added.